extensions/mssql/src/objectManagement/ui/loginDialog.ts

```diff
@@ -155,21 +151,21 @@ export class LoginDialog extends ObjectManagementDialogBase<ObjectManagement.Log
 			this.onObjectValueChange();
 			await this.runValidation(false);
 		});
-		const authTypeContainer = this.createLabelInputContainer(view, localizedConstants.AuthTypeText, this.authTypeDropdown);
+		const authTypeContainer = this.createLabelInputContainer(localizedConstants.AuthTypeText, this.authTypeDropdown);
 
-		this.enabledCheckbox = this.createCheckbox(view, localizedConstants.EnabledText, this.objectInfo.isEnabled);
+		this.enabledCheckbox = this.createCheckbox(localizedConstants.EnabledText, this.objectInfo.isEnabled);
 		this.enabledCheckbox.onChanged(() => {
```

Comment:
Make sure you go through and add any disposables to a disposable store/registration to ensure they're cleaned up properly.